bedrock/thunderbird/templates/thunderbird/features.html

```diff
@@ -0,0 +1,243 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "thunderbird/base-resp.html" %}
+
+{% block page_title %}{{ _('Thunderbird Features') }}{% endblock %}
+
+{% block page_desc %}
+  {% if LANG.startswith('en-') %}
+    Thunderbird makes email better for you, bringing together speed, privacy and latest technologies. Learn more.
```

Comment:
We consistently do *not* use the Oxford comma in these situations, so I think this is fine. It does seem we should maybe add "the" before "latest technologies" though.

(Personally I still like the Oxford comma. :grin:)